packages/server/lib/util/settings.js

```diff
@@ -93,6 +93,10 @@ module.exports = {
     }, _.cloneDeep(obj))
   },
 
+  isComponentTesting (options = {}) {
+    return options.experimentalComponentTesting || options.componentTesting || options.testingType === 'component'
+  },
```

Comment:
We have a lot of flags for CT, apparently... we should reduce these to just one or two eventually.

Reply:
Oy vey. Yeah we need to do this like ASAP.